alcs-frontend/package-lock.json

```diff
@@ -13984,13 +13984,15 @@
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/@csstools/postcss-unset-value/-/postcss-unset-value-1.0.2.tgz",
       "integrity": "sha512-c8J4roPBILnelAsdLr4XOAR/GsTm0GJi4XpcfvoWk3U6KiTCqiFYc63KhRMQQX35jYMp4Ao8Ij9+IZRgMfJp1g==",
-      "dev": true
+      "dev": true,
+      "requires": {}
```

Comment:
Very odd that every time I run npm i it removes the empty requires, and yours adds it back.

Reply:
yes, I noticed that too, but I think we using the same npm version